corehq/form_processor/models.py

```diff
@@ -88,4 +124,15 @@ def read_content(self):
             content = f.read()
         return content
 
+
+class XFormOperationSQL(models.Model):
```

Comment:
Fair enough. We could make the attachments field lazy loaded but that's
probably about the same anyway.